ert/data/record/_record.py

```diff
@@ -241,27 +319,53 @@ def _flatten_record_dict(
 
     @property
     def data(self) -> record_data:
-        # internally, recordtree is represented as BlobRecord; we don't need data here
+        """Internally, RecordTree is represented as BlobRecord; therefore
+        this property shoudn't be used.
+
+        Returns:
+            record_data: no data
```

Comment:
"no data" is technically speaking wrong?

Reply:
Will make up something :)